plugin-dev/Source/Sentry/Sentry.Build.cs

```diff
@@ -161,5 +161,14 @@ public Sentry(ReadOnlyTargetRules Target) : base(Target)
 
 			AddEngineThirdPartyPrivateStaticDependencies(Target, "libcurl");
 		}
+		else
+		{
+			// Exclude sources in `Private/GenericPlatform` that use sentry-native API from the build if target platform isn't supported
+			// Plugin extensions that add support for more platforms (e.g. consoles) can override this define if needed
+			PublicDefinitions.Add("USE_SENTRY_NATIVE=0");
+
+			// TODO: Add a valid link to docs (https://github.com/getsentry/sentry-docs/issues/13861)
+			Console.WriteLine("To use Sentry SDK on game consoles follow the instructions at https://docs.sentry.io/platforms/consoles");
```

Comment:
```suggestion
			if (GetType() == typeof(Sentry))
			{
				Console.WriteLine("To use Sentry SDK on game consoles follow the instructions at https://docs.sentry.io/platforms/consoles");
			}
```

Reply:
We can print this message to logs only when building for certain platform without the corresponding plugin extension being installed.

Example: for Xbox, this type check should return `Sentry_XSX` if the corresponding extension is installed (see [Sentry_XSX.Build.cs](https://github.com/getsentry/sentry-unreal-xbox/blob/4bc6ea5495d388b6dc0b9dcab43b8c9991094b8c/Sentry/Source/Sentry_XSX.Build.cs)). This indicates that the user has already completed middleware verification and gained access to the `sentry-unreal-xbox` repository so there's no point in printing the link.

Reply:
I think it won't if we're testing type. You'd need to check `IsAssignableTo`

`==` will check for exact type being equal